xbmc/utils/StringUtils.h

```diff
@@ -194,3 +194,11 @@ class StringUtils
 private:
   static std::string m_lastUUID;
 };
+
+struct sortstringbyname
+{
+  bool operator()(const std::string& strItem1, const std::string& strItem2)
+  {
+    return StringUtils::CompareNoCase(strItem1, strItem1) < 0;
```

Comment:
The second one should be strItem2, right? Otherwise you're comparing strItem1 with itself.